webapp/src/app/core/modules/openapi/api/activity.serviceInterface.ts

```
@@ -46,4 +47,20 @@
+     * @param badPracticeId 
+     * @param badPracticeFeedback 
+     */
+    provideFeedbackForBadPractice(badPracticeId: number, badPracticeFeedback: BadPracticeFeedback, extraHttpRequestParams?: any): Observable<{}>;
+
+    /**
+     * 
+     * 
+     * @param badPracticeId 
+     * @param state 
+     */
+    resolveBadPractice(badPracticeId: number, state: 'GOOD_PRACTICE' | 'FIXED' | 'CRITICAL_ISSUE' | 'NORMAL_ISSUE' | 'MINOR_ISSUE' | 'WONT_FIX' | 'WRONG', extraHttpRequestParams?: any): Observable<{}>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)